content/riak/kv/2.2.0/using/repair-recovery/rolling-replaces.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Rolling Replaces"
+description: ""
+project: "riak_kv"
+project_version: "2.2.0"
+menu:
+  riak_kv-2.2.0:
+    name: "Rolling Replaces"
+    identifier: "repair_recover_replace"
+    weight: 106
+    parent: "managing_repair_recover"
+toc: true
+---
+
+[upgrade]: /riak/kv/2.2.0/setup/upgrading/cluster/
+[rolling restarts]: /riak/kv/2.2.0/using/repair-recovery/rolling-restart/
+[add node]: /riak/kv/2.2.0/using/cluster-operations/adding-removing-nodes
+
+Riak KV functions as a multi-node system, so cluster-level [version upgrades][upgrade] and [restarts][rolling restarts] can be performed on a node-by-node or *rolling* basis.
+
+The following steps should be undertaken on each Riak KV node that you wish to replace:
+
+1\. Create a free node:
+
+  a\. [Create an additional node][add node] with similar specifications to the other nodes in the cluster.
+
+  b\. Or leave a node that is currently in the cluster:
+
+  ```bash
+  riak-admin cluster leave «nodename»
+  ```
+
+  After creating a node or leaving a node, wait for all transfers to complete:
+
+  ```bash
+  riak-admin transfers
+  ```
+
+2\. Join the free node to your cluster:
+
+```bash
+riak-admin cluster join «nodename_leaving_cluster»
```

Comment:
Perhaps riak-admin cluster join <<free_node>>  This would apply to both scenarios where you've added a free node or you've taken an existing node to leave the cluster as the free node?